src/request-handler.ts

```diff
@@ -202,6 +202,43 @@ export function createRequestHandler(
       body = explicitBodyData;
     }
 
+    const headers = getConfig<HeadersInit>(requestConfig, 'headers') || {};
+
+    // Only applicable for request methods 'PUT' and 'DELETE' without a body
+    if (['DELETE', 'PUT'].includes(method) && !body) {
```

Comment:
Thanks for the first contribution! 🙌

I think that instead of removing the headers explicitly below, we could simply remove the line 59 of this file and set that default Content Type somewhere here but only for requests other than PUT and DELETE, and if user didn't specify the headers explicitly of course.

The reason is that in some implementations of weird, APIs people may want to explicitly specify the Content Type for these kinds of requests anyways, so removing the headers would cause a bug for them.

Additionally that should save us some bundle size.

Reply:
I also think that adding rather than removing is better in terms of avoiding potential bugs. I thought it was a good suggestion, so I made the changes! Please check it out 😎

Reply:
But I have one question.

I’m not very experienced in writing test code. I added a `headers` at line 72 in `request-handler.spec.ts`, but if I don’t add this line, it seems to affect the headers in the test code I added this time. Do you happen to know why